tests/proofs/dexter/dexter-spec.md

```diff
@@ -5,5 +5,8 @@ module DEXTER-SPEC
   claim <k> now 0 => . ... </k>
         <mynow> #Timestamp(0) </mynow>
 
+  claim <k> (now 0 => .) ~> #dexterCode ... </k>
+        <mynow> #Timestamp(0) </mynow>
+
```

Comment:
We can likely add tests for each macro type using `claim <k> PRETTY_DEXTER_CODE ; POP ; POP ; POP => . ... </k>`, which makes it so it's a no-op on the michelson semantics side.

Reply:
Done